src/main/resources/static/initialRender.js

```diff
@@ -0,0 +1,26 @@
+	ReactDOM.render(
```

Comment:
I can't seem to find how this file gets loaded ...
Who triggers it to execute ?